.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Make a player with a gun, magazine, and some ammo.
+2. Try to aim at a wall.
+3. There's no way to increase aim level.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+Ideally also describe *why* you expect it to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Versions and configuration(please complete the following information):**
+ - OS: [e.g. iOS 8 or Windows 10]
+ - Game Version [e.g. ]
```

Comment:
`[e.g., 7645]`

Reply:
Shouldn't version number be the one shown in the main screen? The problem is that I think precompiled experimental versions just show "0.C" – I had like that until I changed AUR package to manually download the git repo instead of master.zip (takes longer to download, but the version is displayed now, assuming 29938).
![screenshot from 2018-08-03 22-39-30](https://user-images.githubusercontent.com/4951331/43664699-180f83ba-976e-11e8-9dba-bf9a212e2d45.png)

Also, please add example for Linux distros, as now it's unknown how should we state it (Linux? Ubuntu? Ubuntu 18.04?). Also what about the architecture? (32-bit vs 64-bit, or more technically speaking x86 vs x86-64 on modern PCs).

Reply:
`Shouldn't version number be the one shown in the main screen?`

Other members might want this, but personally, I prefer [build numbers](http://gorgon.narc.ro:8080/job/Cataclysm-Matrix/changes) so I can more readily check where and when the issue may have arisen. But that's just me.

Reply:
Either is fine, just make clear instructions where/how to check them :)